disko

```diff
@@ -159,8 +175,15 @@ script=$(nixBuild "${libexec_dir}"/cli.nix \
   --argstr mode "$mode" \
   "${nix_args[@]}"
 )
+
+command="$(echo "$script"/bin/*)"
```

Comment:
Does this break using a new version of the cli with an old version of disko?
Ideally we would fallback or warn the user in this case.
Maybe this could become an array?

```suggestion
command=("$(echo "$script"/bin/*")
```

Reply:
Ah wait. The CLI will always bring its own version of the disko modules anway. Than we don't need to care about compat.

Reply:
Ah yes, that is much better, thank you!